content/docs/atom-guide/setup-atom-projects.md

```diff
@@ -1,111 +1,66 @@
 ---
-title: "Setting Up Atom Projects"
-description: "Set up Atom projects using the Open 3D Engine"
+linkTitle: "Setup"
+title: "Setting up Atom Renderer"
+description: "Get started with Atom Renderer with an Open 3D Engine project or AtomSampleViewer, Atom's standalone application."
 toc: true
 weight: 200
 ---
 
-Get to know the Atom Renderer and its features by setting up an Atom project with the Open 3D Engine. This tutorial covers how to set up two Atom projects: **Atom Test** and **Atom Sample Viewer**.
-
-- **Atom Test**: This project contains a collection of levels that demonstrates Atom's features in the O3DE Editor.
-- **Atom Sample Viewer**: This project is a standalone application that is intended for developers and engineers who want to test out Atom's individual features. Although you will not run the O3DE Editor, O3DE is still required to build this project.
+Get started with **Atom Renderer** through **Open 3D Engine (O3DE)** by creating a new *O3DE project* or building **Atom Sample Viewer**, Atom's standalone application. 
 
 ## Prerequisites
-Before setting up the Atom projects, you must have the O3DE engine installed on your computer. Follow the instructions in the [Setup](/docs/welcome-guide/setup) section of the Get Started Guide to set up O3DE.
 
+Before setting up a new O3DE project or Atom Sample Viewer, you must install O3DE on your computer. Follow the instructions in [Setting Up O3DE](/docs/welcome-guide/setup) of the Get Started Guide.
 
-## Setting up the Atom projects
 
-This tutorial uses the following project name and directories in the examples:
+## O3DE Project
```

Comment:
```suggestion
## O3DE project
```
For section headers, use sentence casing. To that end, changed "Project" to "project".